kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/render/AbstractFormRenderer.java

```diff
@@ -434,6 +439,24 @@ protected void processFormLayout(FormInstance topLevelForm,
                                     }
                                     item.setValue((value != null) ? value.toString() : "");
                                     break;
+                                case "util-date":
+                                    if (value != null && value instanceof java.util.Date) {
+                                        if (!field.isShowTime() && value != null && value.toString().length() >= 10) {
+                                            LocalDate utilDate = ((java.util.Date) value).toInstant().atZone(ZoneId.systemDefault()).toLocalDate();
+                                            value = utilDate.toString();
+                                        } else if (value != null && value.toString().length() >= 1) {
+                                            LocalDateTime utilDateTime = ((java.util.Date) value).toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime();
+                                            value = utilDateTime.toString();
+                                        }
+                                        item.setValue((value != null) ? value.toString() : "");
+                                    } else {
+                                        if (field.isShowTime()) {
```

Comment:
I don't have all context on how the value won't be a `Date` here, but it's right to assume that it would be a String. So I'd say that we could just set the `toString` straight into the field field value unless we have a better way to check the format than using a `substring`